### ISSUE TITLE
New convertible icon

### ISSUE THREAD

**Issue by cassidyjames** [elementary/icons#740]
We symlinked `computer-convertible` to `computer-laptop` for now which makes sense, but a new convertible icon might make sense. At first a tent-style Yoga-like configuration made sense, but honestly I think just a screen might work: it covers both foldable and detachable style convertibles.

**Comment by danrabbit** [comment]
Yeah or maybe something with a very small bezels and a thin keyboard kind of surface or ipad pro like

**Comment by cassidyjames** [comment]
@danrabbit that might work. The problem I could see going that direction is that it is doesn't really work as well for the Yoga-style foldables, so I'm not sure if it'd also feel just as off (if not more) than a laptop.

**Comment by danrabbit** [comment]
@cassidyjames I'm not sure about other Yoga models but the 900 reports as a laptop. Do we have any information about what models are reporting this way?

**Comment by cassidyjames** [comment]
It sounds like at least the HP Pavilion 13 and Thinkpad X1 Yoga.

Here's the source for `hostnamed`: https://cgit.freedesktop.org/systemd/systemd/tree/src/hostname/hostnamed.c#n201, but I don't see "convertible" in there... so I'm really not sure.
